package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "scribe-plugin-underliner",
+  "description": "Underline text in scribe based on matched terms",
+  "version": "0.1.0",
+  "homepage": "https://github.com/guardian/scribe-plugin-underliner",
+  "author": {
+    "name": "hmgibson23",
+    "email": "hugo.gibson@yahoo.co.uk"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/guardian/scribe-plugin-underliner"
+  },
+  "bugs": {
+    "url": "https://github.com/guardian/scribe-plugin-underliner/issues"
+  },
+  "licenses": [
+    {
+      "type": "Apache2",
+      "url": "http://www.apache.org/licenses/LICENSE-2.0"
+    }
+  ],
+  "main": "lib/scribe-plugin-underliner",
+  "engines": {
+    "node": ">= 0.10.0"
+  },
+  "scripts": {
+    "test-firefox": "TEST_SERVER_PORT=9090 BROWSER_NAME='firefox' ./run-tests.sh",
+    "test-chrome": "TEST_SERVER_PORT=9090 BROWSER_NAME='chrome' ./run-tests.sh",
+    "test": "npm run test-chrome",
+    "postinstall": "bower install",
+    "build": "browserify src/scribe-plugin-underliner --standalone scribe-plugin-underliner > ./build/scribe-plugin-underliner.js",
+    "build-prod": "npm run build && uglifyjs ./build/scribe-plugin-underliner.js > ./build/scribe-plugin-underliner.min.js",
+    "watch": "watchify -o ./build/scribe-plugin-underliner.js"
+  },
+  "browserify": {
+    "transform": "babelify"
+  },
+  "devDependencies": {
+    "babel": "^4.4.5",
+    "babelify": "^5.0.3",
+    "bower": "^1.3.12",
+    "browserify": "^9.0.3",
+    "chai": "1.9.1",
+    "glob": "^4.1.5",
+    "http-server": "^0.7.1",
+    "lodash": "^2.4.1",
+    "mocha": "^2.0.1",
+    "scribe-test-harness": "^0.0.6",
+    "selenium-webdriver": "^2.44.0",
+    "webdriver-manager": "^1.0.1",
+    "uglify-js": "^2.4.16",
+    "mversion": "^1.3.0",
+    "watchify": "^2.4.0"
+  },
+  "keywords": [],
+  "dependencies": {
+    "lodash.template": "^3.6.2"
```

Comment:
Oh yeah good call.

On 10 November 2015 at 11:57, James Pamplin <notifications@github.com>
wrote:

> In package.json
> <https://github.com/guardian/scribe-plugin-underliner/pull/1#discussion_r44397894>
> :
>
> > +    "browserify": "^9.0.3",
> > +    "chai": "1.9.1",
> > +    "glob": "^4.1.5",
> > +    "http-server": "^0.7.1",
> > +    "lodash": "^2.4.1",
> > +    "mocha": "^2.0.1",
> > +    "scribe-test-harness": "^0.0.6",
> > +    "selenium-webdriver": "^2.44.0",
> > +    "webdriver-manager": "^1.0.1",
> > +    "uglify-js": "^2.4.16",
> > +    "mversion": "^1.3.0",
> > +    "watchify": "^2.4.0"
> > +  },
> > +  "keywords": [],
> > +  "dependencies": {
> > +    "lodash.template": "^3.6.2"
>
> Is this used?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/guardian/scribe-plugin-underliner/pull/1/files#r44397894>
> .
>

-- 

------------------------------
This e-mail and all attachments are confidential and may also be 
privileged. If you are not the named recipient, please notify the sender 
and delete the e-mail and all attachments immediately. Do not disclose the 
contents to another person. You may not use the information for any 
purpose, or store, or copy, it in any way.  Guardian News & Media Limited 
is not liable for any computer viruses or other material transmitted with 
or as part of this e-mail. You should employ virus checking software.
 
Guardian News & Media Limited is a member of Guardian Media Group plc. Registered 
Office: PO Box 68164, Kings Place, 90 York Way, London, N1P 2AP.  Registered 
in England Number 908396